codespell_lib/tests/test_dictionary.py

```diff
@@ -13,6 +13,8 @@
 try:
     import aspell
     speller = aspell.Speller('lang', 'en')
+    spellerGB = aspell.Speller('lang', 'en_GB-ise')
+    speller_array = (speller, spellerGB)
```

Comment:
Dunno if it'd be worth making this a dict rather than a list?
```python
spellers = dict()
for lang in ('en', 'en_GB-ise'):
    spellers[lang] = aspell.Speller('lang', lang)
```

(and then the code further below would obviously become `for sp in spellers.values()`)

Reply:
Sounds good, but first I want see reactions of @peternewman.